leases/management/commands/import_winter_leases_from_csv.py

```diff
@@ -0,0 +1,171 @@
+                    )
+                except MultipleProfilesException as e:
+                    multiple_profiles.append(
+                        (
+                            lease_input.section_id,
+                            lease_input.place_number,
+                            lease_input.name,
+                            ";".join(e.ids),
+                        )
+                    )
+                    continue
+
+                try:
+                    place = WinterStoragePlace.objects.get(
+                        winter_storage_section_id=lease_input.section_id,
+                        number=lease_input.place_number,
+                    )
+
+                    boat = None
+                    if lease_input.boat_width and lease_input.boat_length:
+                        boat, _created = Boat.objects.get_or_create(
+                            owner=customer_profile,
+                            boat_type=OTHER_BOAT_TYPE,
+                            width=lease_input.boat_width.replace(",", "."),
+                            length=lease_input.boat_length.replace(",", "."),
+                            defaults={"registration_number": lease_input.boat_register},
+                        )
+                    comment = ""
+                    if lease_input.comment:
+                        comment = f"{lease_input.comment}\n"
+                    comment += f"Lease imported on {today().date()}"
+
+                    lease = WinterStorageLease.objects.create(
+                        customer=customer_profile,
+                        place=place,
+                        boat=boat,
+                        start_date=date(day=15, month=9, year=2020),
+                        end_date=date(day=10, month=6, year=2021),
+                        status=LeaseStatus.PAID,
+                        comment=comment,
+                    )
+                    successful.append(
+                        (
+                            str(lease.id),
+                            lease_input.section_id,
+                            lease_input.place_number,
+                            lease_input.name,
+                        )
+                    )
+                except (WinterStoragePlace.DoesNotExist, ValidationError) as e:
+                    failed.append(
+                        (
+                            lease_input.section_id,
+                            lease_input.place_number,
+                            lease_input.name,
+                            str(e),
+                        )
+                    )
+
+        with open("./successful_leases.csv", "w+", encoding="utf-8") as successful_file:
+            writer = csv.writer(successful_file, delimiter=",")
+            writer.writerow(["Lease id", "Section id", "Place number", "Customer name"])
+            writer.writerows(successful)
+
+        with open("./multiple_profiles.csv", "w+", encoding="utf-8") as multiple_file:
```

Comment:
What's the plan for processing multiple_profiles.csv and failed_leases.csv?
Should the admins go through them manually? If so, should these files have more information about the failed lease? Or line number in original file, in case admins want to look up information from there?

Reply:
Not on the first place. Ideally, we'd get those files and see if there's anything that we can fix manually. If there's anything regarding the customer information, the we hand it to the admins.